kubernetes/loculus/templates/silo-deployment.yaml

```diff
@@ -65,15 +65,45 @@ spec:
             periodSeconds: 10
             failureThreshold: 3
             timeoutSeconds: 5
+        - name: silo-importer
```

Comment:
Anya's thoughts: maybe we shouldn't split this up but have the silo-import job also run silo preprocessing - this way we don't need to make the bash script aware of the sentinel file structure and can do all that in one place.

Reply:
That's fine but then you'll need to compile Silo here

Reply:
yeah sounds good to keep like this for now